webpack.config.js

```diff
@@ -41,6 +41,10 @@ const shouldCheckForCycles = process.env.CHECK_CYCLES === 'true';
 const codeSplit = config.isEnabled( 'code-splitting' );
 const isCalypsoClient = process.env.CALYPSO_CLIENT === 'true';
 
+const workerCount = process.env.WORKERS
+	? Math.floor( +process.env.WORKERS / 2 )
```

Comment:
if it's spec'd, do we still want to halve it? maybe just limit it to os.cpus().length?

Reply:
Very much work in progress. 

The thinking is that the user declares workers, we have two webpack bits that do parallel work: thread loader and terser. We need to divide the total workers amongst the bits that may soon up multiple workers.

6 workers among 2 parallel jobs: 3 each. 

I'm not sure how concurrent these webpack parts run, if they run in series, each can use all the workers. If not, we need to make sure the total is not greater than the provided value.

Reply:
My assumptions about about thread-loader and terser potentially using more workers than was provided seems to be incorrect. I've change to respect whatever value is provided.